src/amuse/community/brutus/Makefile

```diff
@@ -9,9 +9,9 @@ AMUSE_DIR?=../../../..
 # mpfr: http://www.mpfr.org/
 # mpfrc++: http://www.holoborodko.com/pavel/mpfr/
 # Change these paths in this Makefile and in src/makefile
-GMP_LIBS   ?= -L/data1/boekholt/Packages/mpreal_local/lib/ -lgmp  # libgmp.a
-MPFR_LIBS  ?= -L/data1/boekholt/Packages/mpreal_local/lib/ -lmpfr # libmpfr.a
-MPFR_FLAGS ?= -I/data1/boekholt/Packages/mpreal_local/include/    # mpreal.h
+GMP_LIBS   = -L/usr/lib/x86_64-linux-gnu/ -lgmp  # libgmp.a
+MPFR_LIBS  = -L/usr/lib/x86_64-linux-gnu/ -lmpfr # libmpfr.a
+MPFR_FLAGS = -I./mpfrc++    # mpreal.h
```

Comment:
these should be picked up by the amuse config, so overriding those can break the compilation - why was this necessary?

Reply:
My request was to use use the interface.cc and Interface.py. The makefile is not of interest. changes in the makefile was to get it work on my machine. But need not be relevant for the master. I did not know that I can't choose the files for pull request sepaerartely.

Reply:
ok, no problem will fix it..
I don't think you can select files from a pull request, usually I make sure I only commit relevant files in a seperate branch, eg:
```
git checkout master
git branch brutus_mpreal
git checkout brutus_mpreal
git add <file1> <file2> etc
git commit
git push <your remote> brutus_mpreal:brutus_mpreal
```
(a seperate branch is handy since it will allow you to work on multiple things at the same time)